src/main/java/org/broadinstitute/hellbender/utils/dataflow/BucketUtils.java

```diff
@@ -45,8 +44,12 @@ public static boolean isHadoopUrl(String path) {
      */
     public static InputStream openFile(String path, PipelineOptions popts) {
         try {
+            if (null==path) throw new NullPointerException("path was null, shouldn't be.");
+            if (null==popts) throw new NullPointerException("popts was null, shouldn't be.");
             if (BucketUtils.isCloudStorageUrl(path)) {
-                return Channels.newInputStream(new GcsUtil.GcsUtilFactory().create(popts).open(GcsPath.fromUri(path)));
+                GcsPath gcsPath = GcsPath.fromUri(path);
+                if (null==gcsPath) throw new NullPointerException("parsing path yielded null, shouldn't have.");
+                return Channels.newInputStream(new GcsUtil.GcsUtilFactory().create(popts).open(gcsPath));
             } else {
                 return new FileInputStream(path);
             }
```

Comment:
I don't want exceptions that aren't actually the users fault to be labeled as UserException.  Things like null function parameters are more likely to be caused by some internal bug, since user input should be checked for basic properties at the command line.